### ISSUE TITLE
document that NFS temporary directories don't (currently) work

### ISSUE THREAD

**Issue by jairideout** [qiime2/docs#278]
**Bug Description**
TL;DR: There's [a bug](https://bugs.python.org/issue24564) in Python 3.4, 3.5, and 3.6 where `shutil.copytree` doesn't work when copying from NFS to NFS (even if it is the same mount). This means that users can't use an NFS directory as their temporary directory (e.g. `TMPDIR`).

**Comments**
1. This isn't something we can fix as we're waiting on upstream Python 3 to fix the bug in a release. For now, this issue tracks the solution/workaround (i.e. don't use NFS for temporary directories). It'd also be nice to add a note somewhere in the docs about this temporary directory issue.
2. **Note:** before writing up the docs, check whether the bug has been fixed in a Python release that's distributed in the QIIME 2 environment files (then there's no need to document the workaround).

**References**
See the forum topic for details: [forum x-ref](https://forum.qiime2.org/t/demux-emp-single-error-in-moving-pictures-tutorial/2701/7?u=jairideout)
